wgsl/index.bs

```diff
@@ -430,15 +430,25 @@ A <dfn noexport>block comment</dfn> is a kind of [=comment=] consisting of:
 * Then any sequence of:
     * A [=block comment=], or
     * Text that does not contain either `*/` or `/*`
-* Then either:
-    * The two characters `*/`, or
-    * The end of the source text.
+* Then the two characters `*/`
 
 Note: Block comments can be nested.
 Since a block comment requires matching start and end text sequences, and allows arbitrary nesting,
 a block comment cannot be recognized with a regular expression.
 This is a consequence of the Pumping Lemma for Regular Languages.
 
+<div class='example' heading='Example comments'>
```

Comment:
If possible please make this 'example wgsl' so that the example is tested. (Due to past constraints we only test such examples.)